### PR TITLE
Update data source documentation

### PR DESCRIPTION
**Pull request by joshuayhwu**
Fixes #{issue number here}

  - [ ] Up to date with `dev` branch
  - [ ] Branch name follows [guidelines](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md#feature-branching)
  - [ ] All PR Status checks are successful
  - [ ] Peer reviewed and approved

Any questions? See the [getting started guide](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md)

### CODE REVIEW

File: misc/DataSources.md [hunk 1]
```diff
@@ -2,30 +2,205 @@
 
 The following is a list of Los Angeles datasets that could be used in combination with or in addition to 311 Data for analysis that would be relevant to Neighborhood Councils.
 
+## LA 311 Data
```
Comment (nichhk) [inline misc/DataSources.md:5]: Could you move this section to the top, before "# List of additional datasets for comparisons"?

File: misc/DataSources.md [hunk 2]
```diff
@@ -2,30 +2,205 @@
...
+* [Los Angeles 311 Data by Year](https://data.lacity.org/City-Infrastructure-Service-Requests/MyLA311-Service-Request-Data-2022/i5ke-k6by)
```
Comment (nichhk) [inline misc/DataSources.md:6]: To clarify, this is only for 2022, right?

**Review by nichhk — APPROVED** [review]
Thanks Josh! This is super helpful. Two small comments. Please merge it after fixing them.

File: misc/DataSources.md (hunk 1 above)
Comment (joshuayhwu) [reply misc/DataSources.md:1]: Resolved

File: misc/DataSources.md (hunk 2 above)
Comment (joshuayhwu) [reply misc/DataSources.md]: Yes. I have updated the data name to just 2022.

**Pull request merged**